docs/proposals/proposals.md

```diff
@@ -16,3 +16,4 @@ This folder includes all the Venice Improvement Proposals. Click to read further
 | [VIP-1](vip-1.md)                                    | Authentication Service API                                                     | Accepted         |
 | [VIP-2](https://github.com/linkedin/venice/pull/513) | Removing Per Record Offset Metadata From Venice-Server Storage With Heartbeats | Under Discussion |
 | [VIP-3](vip-3.md)                                    | Rust Server Read Path                                                          | Under Discussion |
+| [VIP-4](vip-4.md)                                    | Store Lifecycle Hooks                                                          | Under Discussion |
```

Comment:
Isn't it more about `Store Version Lifecycle` rather than `Store Lifecycle`?

Reply:
In the current proposal, it is both, and I consider store-version lifecycle to be a subset of the store lifecycle.

Schema registration is a global event which I think we may want to emit monitoring events for, so that is why I included it.

Reply:
Sounds good.